src/Simulation/qdk_sim_rs/Cargo.lock

```diff
@@ -0,0 +1,1212 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I think this file is usually added to the gitignore, since it is generated by the build.

Reply:
Depends on if we build with `--locked`. That would be great to do for reproducible builds, but that's unfortunately not really consistent with generating Cargo.toml on the fly.